pyproject.toml

```diff
@@ -0,0 +1,40 @@
+[build-system]
+requires = [
+  "setuptools >= 40.9.0",
+  "wheel",
+]
+build-backend = "setuptools.build_meta"
```

Comment:
Could you detail why we need this new `[build-system]` section?

Reply:
Just a test to get the CI builds running. Locally everything works so I need to debug on CI. From the failing tests I get that this is not the issue. Trying `--no-build-isolation` next as you suggested offline.